config/initializers/filter_parameter_logging.rb

```diff
@@ -3,4 +3,6 @@
 # Be sure to restart your server when you modify this file.
 
 # Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password]
+Rails.application.config.filter_parameters += %i[
+  password passw secret token _key crypt salt certificate otp ssn
```

Comment:
the new default file from rails omitted `password`.  i wasn't sure if that was already a default in `filter_parameters`, so i added it here to be safe, but happy to remove if it's redundant.